apps/node/dal.js

```diff
@@ -0,0 +1,12 @@
+data = {};
```

Comment:
```suggestion
const data = {};
```

You almost never need "real" globals in Node, since all node modules are singletons (only exception is if your module has to coexist with other versions of itself). Declare data as `const` here, if it's scoped to the module then every caller will get the same instance.

Also, if you want global stuff you can do `global.myProp`. This code would fail in strict mode (which you can opt into by adding `'use strict';` to the top of files if you want)

Reply:
👍